frontend/src/components/session-in-progress/question/question.tsx

```diff
@@ -38,16 +50,14 @@ const Question = (props: QuestionProps): ReactElement => {
                 </div>
 
                 <div className="answer-info">
-                  <p className="answer-text">{answer}</p>
+                  <p className="answer-text">{answer.text}</p>
 
                   {props.showPercentages ? (
                     <>
                       <div
                         style={{ width: `${percentages[index]}%` }}
```

Comment:
This still seems to be mocked out. We can leave it in for now but we should switch it in the future.